client/stylesheets/_toCat.scss

```diff
@@ -3802,12 +3802,13 @@ On click the images will be displayed at normal size to complete the effect
     padding: 10px 10px;
     background-color: #f8504b;
     text-align: center;
+    border-top: 3px solid #ee4641;
 }
 
 .footerDivide {
-    border-top: 1px solid black;
+    border-top: 1px solid #404040;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border-bottom, border-top, padding-bottom, padding-top